ui/main/src/app/api/opfab.api.ts

```
@@ -0,0 +1,54 @@
+
+    private static initAlertAPI() {
+        opfab.alertMessage = {
+            messageLevel: Object.freeze(MessageLevel),
+            show(message, messageLevel) {
+                const msg = new Message(message, messageLevel);
+                return AlertMessageService.sendAlertMessage(msg);
+            }
+        };
+        Object.freeze(opfab.alertMessage);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)